asset/script/main.js

```diff
@@ -1,15 +1,18 @@
 const mobileMenu = document.querySelector('.mobile-menu');
 const hamburgar = document.getElementById('hamburger-icon');
 const navClose = document.querySelectorAll('.mobile-nav-link');
+const brand = document.querySelector('#brand');
 
 function toggleMobileMenu() {
   mobileMenu.classList.toggle('open');
   hamburgar.classList.toggle('change');
+  brand.classList.toggle('closeBrand');
 }
 
 function closeNav() {
   hamburgar.classList.remove('change');
   mobileMenu.classList.remove('open');
+  brand.classList.remove('closeBrand');
 }
```

Comment:
Use ES6 syntax for your funtions by using arrow function. eg `const closeNav = () => {
.
.
.
}`